### PR TITLE
#77_fix_daterange_for_logs

### PR DESCRIPTION
**Pull request by vuongc**
Fix issue about dateRange for logs + delete attribute from object if string length is 0 in logs routes

### CODE REVIEW

File: webapp/app/components/DatePicker/index.js [hunk 1]
```diff
@@ -30,16 +30,22 @@ export default class DatePicker extends React.Component {
         this.props.onChange(this.props.value);
       }
     } else {
-      this.setState({ date: moment().toString() });
+      this.setState({ date: moment().toDate().toString() });
```
Comment (clairealmozinos) [inline webapp/app/components/DatePicker/index.js:33]: @vuongc why did you had a toDate()?

**Comment by vuongc** [comment]
@clairealmozinos Because I wanted to know the timezone type to compared it with the date from the datepicker:

``` shell
moment().toString() => Tue Jul 19 2016 12:14:36 GMT+0200
moment().toDate().toString() => Tue Jul 19 2016 12:14:36 GMT+0200 (CEST)
new Date().toString() => Tue Jul 19 2016 12:14:36 GMT+0200 (CEST)
```

**Comment by clairealmozinos** [comment]
@vuongc did you verify that it's still working in other places where the component is used?

**Comment by vuongc** [comment]
@clairealmozinos I didn't do it, I will do it and come back to you afterward

**Pull request merged**